models/apartments.rb

```diff
@@ -0,0 +1,18 @@
+class Apartment < ActiveRecord::Base
+  has_many :tenants
+  # attr_accessor :renters, :rent
+  # attr_reader :address, :sqft, :num_beds, :num_baths
+  # def initialize (address,rent,sqft,num_beds,num_baths,renters)
+  #   @address = address
+  #   @rent = rent
+  #   @sqft = sqft
+  #   @num_beds = num_beds
+  #   @num_baths = num_baths
+  #   @renters = renters
+  # end
+  # def add_tenant (tenant)
+  #   if @renters.length < @num_beds
+  #     @renters << tenant
+  #   end
+  # end
```

Comment:
That's what I wasn't sure about-- if I could define class methods in addition to AR methods. Based on your comment, it sounds like you can. So you're saying that an `add_tenant` method should be accomplished by updating the apartment attribute for a tenant in AR, rather than defining the method in the ruby class, whereas something as trivial as concatenating first and last name should be done in the ruby class out of convenience... right?